src/PKSim.Core/Snapshots/Mappers/CompoundPropertiesMapper.cs

```diff
@@ -199,6 +199,19 @@ private void updateAlternativeSelection(CompoundGroupSelection snapshotCompoundG
             return;
          }
 
+         var alternativeGroup = compoundProperties.Compound.ParameterAlternativeGroup(snapshotCompoundGroupSelection.GroupName);
+         if (alternativeGroup == null)
+         {
+            _logger.AddWarning(PKSimConstants.Error.CompoundGroupNotFoundFor(snapshotCompoundGroupSelection.GroupName, compoundProperties.Compound.Name));
+            return;
+         }
+
+         var alternative = alternativeGroup.AlternativeByName(snapshotCompoundGroupSelection.AlternativeName);
+         if (alternative == null)
+         {
+            _logger.AddWarning(PKSimConstants.Error.CompoundAlternativeNotFoundFor(snapshotCompoundGroupSelection.AlternativeName, alternativeGroup.DefaultAlternative?.Name, snapshotCompoundGroupSelection.GroupName, compoundProperties.Compound.Name));
```

Comment:
Instead of using an alternative that's does not exist, default alternative will be used (e.g. Simulation created will be totally different but well shit in shit out)

Reply:
I would make it an **error**.  Think that's ok to create a simulation anyway and give the user a possibility to adjust it later, but missing alternative should not be just a warning.

Reply:
What about missing parameters? Isn't is as critical?
Alternatively, we could downgrade all errors and warnings
e.g. All AddWarning become AddInfo
And Missing parameters. missing processes etc.. become AddWarning?